src/Configuration/GeneratorConfiguration.php

```diff
@@ -107,9 +120,26 @@ public function getDefaultGroupCombinations(ClassToGenerate $classToGenerate): a
         }, $this->defaultGroupCombinations);
     }
 
+    public function shouldAssignUnknownArrays(): bool
```

Comment:
can we phpdoc this? something like this:

```suggestion
    /**
     * If this is false, arrays with sub type PropertyTypeUnknown are treated as error.
     * If this is true, deserialize assigns the raw array and serialize just takes the raw content of the field.
     */
    public function shouldAssignUnknownArrays(): bool
```

Reply:
Added via 0075dd75636f3a79ed0148c5105754bd025946c3